facebook/OpenBIC
# fby3.5: hd: Add FM throttle

Summary:
- Set TSI high temperature threshold to 95 degree C.
- If TSI trigger TempHighAlert, record FM_throttle event log.

Dependency: #593 

Test Plan:
- Build code: PASS
- Check SEL: PASS

Log:
root@bmc-oob:~# log-util slot1 --print
2018 Mar 10 07:39:54 log-util: User cleared FRU: 1 logs
1    slot1    2018-03-10 07:41:34    ipmid            SEL Entry: FRU: 1, Record: Standard (0x02), Time: 2018-03-10 07:41:34, Sensor: SYSTEM_STATUS (0x10), Event Data: (10FFFF) FM_Throttle throttle Assertion
1    slot1    2018-03-10 07:41:36    sensord          ASSERT: Upper Critical threshold - raised - FRU: 1, num: 0x4 curr_val: 95.00 C, thresh_val: 95.00 C, snr: MB_SOC_CPU_TEMP_C
1    slot1    2018-03-10 07:42:00    sensord          DEASSERT: Upper Critical threshold - settled - FRU: 1, num: 0x4 curr_val: 93.88 C, thresh_val: 95.00 C, snr: MB_SOC_CPU_TEMP_C
1    slot1    2018-03-10 07:42:04    ipmid            SEL Entry: FRU: 1, Record: Standard (0x02), Time: 2018-03-10 07:42:04, Sensor: SYSTEM_STATUS (0x10), Event Data: (10FFFF) FM_Throttle throttle Deassertion
